core/src/main/java/jenkins/tools/GlobalToolConfiguration.java

```diff
@@ -70,7 +70,7 @@ public String getUrlName() {
 
     @Override
     public Permission getRequiredPermission() {
-        return Jenkins.ADMINISTER;
+        return Jenkins.SYSTEM_READ;
```

Comment:
I have a doubt. The author of a plugin contributing with a global tool configuration might be considered his/her piece of UI is not going to be shown to anyone else but Administers. So s/he didn't need to check the permission because it was already done. Now, everyone with `SYSTEM_READ` could see his/her piece of UI, something unexpected. 

It's not a good practice to not check the permission required for your piece of UI, but possible. In this circumstances this change may imply a security concern because the plugin shows something sensitive or allows to modify the state without submitting the form with the global _save_ button.

WDYT?

Reply:
What part of your feedback is new to this PR, rather than inherent to the entire SystemRead JEP / the initial PR? Seems like that's basically the entire idea – you see the configuration, but cannot change it.

Reply:
How is this any different to the global configure page?

Reply:
It's the same situation. Just wondering whether it's worth disabling all fields in the page, without having to change the plugin to check the `readOnlyMode`. For example via javascript or something similar.

Reply:
this already does this, the only case where it's not done is in custom jelly taglibs in a plugin or plain html elements.

There's a video showing this here:
https://www.youtube.com/watch?v=7nugAEhXOmM&feature=youtu.be
(there was an issue with the recording so the intro and presentation are missing)

Start here if you want to see me installing a plugin which didn't have to change anything for its' UI:
https://youtu.be/7nugAEhXOmM?t=738